### DISCUSSION TITLE
Hide/Unhide option

### DISCUSSION THREAD

**Discussion by asaf2298** [music-assistant/support#1119] [Feature requests and ideas]
Music Assistants is really great.
 It could be nice if there will be an option to add to the main page (3 dots) an option to hide/unhide folders that you might find unnecessary (for me it’s browser and tracks and maybe more for different users in the future updates).
I need the UI to be as easy as it gets for some house members.

**Comment by OzGav** [comment]
Please have a look and see if either of these discussions are what you are asking. If so please continue there.

https://github.com/music-assistant/hass-music-assistant/discussions/435
https://github.com/music-assistant/hass-music-assistant/discussions/1008

**Reply by asaf2298** [reply]
Amazing, I didn't saw these posts. this leaves only the second request.

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
